third_party/crashpad/crashpad/util/net/http_transport_socket.cc

```diff
@@ -329,7 +329,8 @@ base::ScopedFD CreateSocket(const std::string& hostname,
         } else if (WaitUntilSocketIsReady(result.get())) {
           return result;
         }
-        return base::ScopedFD();
+        continue;
```

Comment:
I'm kind of surprised no other platform (that I'm aware of) has experienced this issue. I wonder if the `socket` system call above is expected to fail if the platform doesn't support IPv6?

Reply:
@hlwarriner:
Thks for your reply.
We did not know other platform situation.
We also have other platform(also linux platform but use yocto project) which can pass the case defaultly.
Checked the log, find that its first member from the addrinfo is ipv4. so it can pass.

BTW,from the function logic, it should traverse the list, to find the available ap.
Thks~

Reply:
Thanks. I posted another question about this in https://b.corp.google.com/u/0/issues/323792488. Can you please take a look?